1.26.0

matteobruni opened the pull request:
feat: particle when destroyed can be splitted, closes #994

matteobruni committed:
feat: particle when destroyed can be splitted, closes #994

ghost commented:
<img src="https://www.deepcode.ai/icons/green_check.svg" width= "50px" align= "left"/> Congratulations :tada:. DeepCode [analyzed](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/2cc450673dc021f84bf4afc074011af11730834c/matteobruni/tsparticles/a5b844a6f2f104bf08e1f8ae4946a01e081f4a1e/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=0&) your code in 15.057 seconds and we found no issues. Enjoy a moment of no bugs :sunny:.

#### 👉 View analysis in [**DeepCode’s Dashboard**](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/2cc450673dc021f84bf4afc074011af11730834c/matteobruni/tsparticles/a5b844a6f2f104bf08e1f8ae4946a01e081f4a1e/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=0&) | [_Configure the bot_](https://www.deepcode.ai/app/gh/?ownerconfig=matteobruni)

matteobruni committed:
build: code improved

matteobruni committed:
build: code imports fixed

matteobruni committed:
build: more code improved

matteobruni committed:
build: updated lint rules

matteobruni committed:
build: restored broken preact/react webpack configuration

matteobruni committed:
build: updates

matteobruni committed:
feat: particle init on shape drawers

matteobruni committed:
build: converted to class drawers all external shapes

matteobruni committed:
build: updated spiral and its demo

matteobruni committed:
build: fixed some imports

matteobruni committed:
build: fixed imports

matteobruni committed:
fix: fixed rounded rect shape

matteobruni committed:
chore(release): published new version

 - ng-particles@2.9.0
 - inferno-particles@1.9.0
 - jquery-particles@1.26.0
 - preact-particles@1.26.0
 - react-tsparticles@1.26.0
 - svelte-particles@1.9.0
 - particles.vue@2.9.0
 - particles.vue3@1.9.0
 - tsparticles-editor@1.26.0
 - tsparticles@1.26.0
 - ng-particles-demo@1.11.0
 - editor-demo@1.11.0
 - inferno-particles-demo@1.9.0
 - jquery-particles-demo@1.11.0
 - tsparticles-demo@1.11.0
 - preact-particles-demo@1.11.0
 - react-tsparticles-demo@1.11.0
 - svelte-demo@1.9.0
 - particles.vue-demo@1.11.0
 - particles.vue3-demo@1.9.0
 - tsparticles-preset-60fps@1.11.0
 - tsparticles-preset-background-mask@1.11.0
 - tsparticles-preset-basic@1.11.0
 - tsparticles-preset-big-circles@1.11.0
 - tsparticles-preset-bouncing@1.11.0
 - tsparticles-preset-bubbles@1.11.0
 - tsparticles-preset-fire@1.11.0
 - tsparticles-preset-font-awesome@1.11.0
 - tsparticles-preset-fountain@1.11.0
 - tsparticles-preset-sea-anemone@1.11.0
 - tsparticles-preset-snow@1.11.0
 - tsparticles-preset-stars@1.11.0
 - tsparticles-shape-bubble@1.11.0
 - tsparticles-shape-confetti@1.11.0
 - tsparticles-shape-heart@1.11.0
 - tsparticles-shape-multiline-text@1.11.0
 - tsparticles-shape-rounded-rect@1.11.0
 - tsparticles-shape-spiral@1.11.0
 - cra-template-particles-typescript@1.9.0
 - cra-template-particles@1.9.0
 - tsparticles-website@1.26.0
